.github/workflows/createAndTestBetaReleaseBranch.yml

```diff
@@ -0,0 +1,81 @@
+name: Create and Test Beta Release Branch
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create_branch:
+    name: 'Create Branch'
+    runs-on: ubuntu-latest
```

Comment:
looks like this will be tested only on Ubuntu. Any reason to omit Windows?

Reply:
buildAndTest workflow tests linux and windows, this is just for the create branch job

Reply:
This job 'create_branch' is only for branch creation. Testing is in the job run_build_and_test and e2e